src/test/fuzz/tx_pool.cpp

```diff
@@ -230,7 +239,10 @@ FUZZ_TARGET_INIT(tx_pool_standard, initialize_tx_pool)
         Assert(it->second.m_result_type == MempoolAcceptResult::ResultType::VALID ||
                it->second.m_result_type == MempoolAcceptResult::ResultType::INVALID);
 
-        const auto res = WITH_LOCK(::cs_main, return AcceptToMemoryPool(tx_pool, chainstate, tx, GetTime(), bypass_limits, /* test_accept= */ false));
+        DummyChainState& dummyChainState{ static_cast<DummyChainState&>(node.chainman->ActiveChainstate())};
```

Comment:
there is already a `chainstate` symbol, so please use that. Also below it would be good to pass the same symbol to atmp, not the dummy one time and the "real" another time.

Reply:
Done in 83b3697 . The same symbol is being passed to ATMP.